src/pages/authentication/forgotPassword/forgotForm.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Field, reduxForm } from 'redux-form';
 import { RaisedButton } from 'material-ui';
-import { regex } from '../../../utils';
+
+import { regex } from 'utils/';
```

Comment:
Can you remove '/' from 'utils/'

Reply:
We shouldn't, this is an absolute import that I've just added, we should do this to make sure that someones won't take this as an npm package